src/components/checkbox/checkbox.spec.js

```diff
@@ -246,5 +247,96 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
       expect(checkbox.hasClass('ng-invalid')).toBe(true);
     });
+
+    describe('with the md-indeterminant attribute', function() {
+
+      it('should set md-indeterminant attr to true by default', function() {
+        var element = compileAndLink(
+            '<div>' +
+            '<md-checkbox md-indeterminant></md-checkbox>' +
+            '</div>');
+
+        var checkbox = element.find('md-checkbox');
+
+        pageScope.$apply();
+
+        expect(checkbox.hasClass(INDETERMINATE_CSS)).toBe(true);
```

Comment:
Is `toHaveClass` copied over from our branch? This would be a lot easier to read as:

`expect(checkbox).toHaveClass(INDETERMINATE_CSS);`